kyo-core/shared/src/main/scala/kyo/KyoApp.scala

```diff
@@ -5,11 +5,22 @@ import scala.concurrent.duration.Duration
 import scala.util.Try
 
 abstract class KyoApp extends KyoApp.Base[KyoApp.Effects]:
+    val log: Logs.Unsafe = Logs.unsafe
+    val random: Random   = Random.default
+    val clock: Clock     = Clock.default
```

Comment:
There are sometimes issues with class initialization in app classes. I think it's safer to keep these as `def`.

Reply:
Done.